R/navs.R

```diff
@@ -52,10 +52,7 @@ navset_card_pill <- function(
 
   items <- collect_nav_items(..., wrapper = wrapper)
```

Comment:
Note to self: we should probably be differentiating between children and attributes and sending attributes to `card()`. The other problem we might have here is that we should probably expose `padding` and `gap` options for the card we're creating but this is complicated by the fact that users can give `padding` and `gap` etc to `nav_panel()`.